code/classes/DMSSessionBackend.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/**
+ * Class DMSSessionBackend represents a session based DMSDocumentCart backend
+ */
+class DMSSessionBackend extends Object implements DMSCartBackendInterface
+{
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
No need to do this unless you add anything new to the doc block - it's inherited by default. Also remove the empty line on line 8 please